build.sh

```diff
@@ -32,5 +32,16 @@ if [[ -n "${SUFFIX}" ]]; then
 	SUFFIX="-${SUFFIX}"
 fi
 
-docker build --build-arg ARCH="${ARCH}" --build-arg MICROARCH="${MICROARCH}" --build-arg BOOTSTRAP="${BOOTSTRAP}" --build-arg SUFFIX="${SUFFIX}"  -t "${ORG}/${TARGET}:${VERSION}" -f "${NAME}.Dockerfile" .
+# Load local mirror as DIST
+DISTBUILDARG=''
+if [[ -e /etc/portage/make.conf ]]; then
+	export DIST=`portageq envvar GENTOO_MIRRORS|cut -d '"' -f 2|cut -d ' ' -f1`
+	if [ ${DIST} ]; then
+		export DIST="${DIST}/releases/${ARCH}/autobuilds"
+		echo "DIST: using locally preferred mirror: '${DIST}'."
+	fi
+	DISTBUILDARG="--build-arg DIST=${DIST}"
```

Comment:
why not always pass the `--build-arg` and get a default on `DIST` on line 36? we could get rid of `DISTBUILDARG`